packages/block-library/src/search/index.php

```diff
@@ -90,6 +94,11 @@ function render_block_core_search( $attributes ) {
 			$input->set_attribute( 'aria-hidden', 'true' );
 			$input->set_attribute( 'tabindex', '-1' );
 		}
+		// Instant search is only available when using the enhanced pagination.
+		if ( $enhanced_pagination ) {
+			$input->set_attribute( 'data-wp-bind--value', 'state.search' );
+			$input->set_attribute( 'data-wp-on--input', 'actions.updateSearch' );
```

Comment:
How about on-async here?
```suggestion
			$input->set_attribute( 'data-wp-on-async--input', 'actions.updateSearch' );
```

Reply:
yeah, that makes sense of course! 🙂

Reply:
done in 9a4e78f48985973d79128119c626c06947a836b1